scripts/funcotator/data_sources/getGencode.sh

```diff
@@ -158,25 +162,35 @@ function getGencodeFiles()
 
     mkdir -p ${OUT_DIR_NAME}/${refVersion}
     pushd ${OUT_DIR_NAME}/${refVersion} &> /dev/null
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
 
-    gunzip gencode.v${version}.annotation.gtf.gz
-    gunzip gencode.v${version}.pc_transcripts.fa.gz
+		fileRefVersion=${version}
+		if [[ "${refVersion}" == "hg38" ]] ; then
+			sourceUrl=ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
+			wget ${sourceUrl} 
+    	wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
+		else
+			fileRefVersion="${version}lift37"
+			sourceUrl=ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/GRCh37_mapping/gencode.v${fileRefVersion}.annotation.gtf.gz
+			wget ${sourceUrl}
+			wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/GRCh37_mapping/gencode.v${fileRefVersion}.pc_transcripts.fa.gz
+		fi
+		
+    gunzip gencode.v${fileRefVersion}.annotation.gtf.gz
+    gunzip gencode.v${fileRefVersion}.pc_transcripts.fa.gz
 
     # We must fix the information in the gencode gtf file:
     echo "Reordering Gencode GTF data ..."
-    ${SCRIPTDIR}/fixGencodeOrdering.py gencode.v${version}.annotation.gtf > gencode.v${version}.annotation.REORDERED.gtf
+    ${SCRIPTDIR}/fixGencodeOrdering.py gencode.v${fileRefVersion}.annotation.gtf | sed -e '1,5s$^#\([a-z]\)$##\1$' > gencode.v${fileRefVersion}.annotation.REORDERED.gtf
```

Comment:
Are you guaranteed to have exactly 5 lines of header?

Reply:
By the spec, no, but in practice this is what I've always seen.

This is really just applying that sed expression over the first 5 lines anyway and if those lines have single and not double comments it adds a second `#` in there.  I'll expand the range on the `sed` command to be safe (and it won't affect lines that aren't commented).